### PR TITLE
[bitnami/redis] Add relabelings to ServiceMonitor

### PR DESCRIPTION
**Pull request by Vaproc**
**Description of the change**

Allows relabeling labels for exported redis metrics via ServiceMonitor (before, only metricRelabelings was supported)

**Benefits**

Additional configuration options

**Possible drawbacks**

None

**Applicable issues**

  - related #7330

**Additional information**

None

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/redis/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 15.0.3
+version: 15.1.3
```
Comment (migruiz4) [inline bitnami/redis/Chart.yaml:28]:
```suggestion
version: 15.1.0
```

File: bitnami/redis/README.md [hunk 2]
```diff
@@ -388,6 +388,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `metrics.serviceMonitor.interval`            | The interval at which metrics should be scraped                                                  | `30s`                    |
 | `metrics.serviceMonitor.scrapeTimeout`       | The timeout after which the scrape is ended                                                      | `""`                     |
 | `metrics.serviceMonitor.relabellings`        | Metrics relabellings to add to the scrape endpoint                                               | `[]`                     |
+| `metrics.serviceMonitor.metricRelabelings`   | Metrics relabellings to add to the scrape endpoint                                               | `[]`                     |
```
Comment (migruiz4) [inline bitnami/redis/README.md:391]: Could you please change the description of both relabellings* (just noticed the value has a typo) and metricRelabelings to something that helped differentiate both settings?

From the prometheus-operator CRD: https://github.com/prometheus-operator/prometheus-operator/blob/master/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml

```suggestion
| `metrics.serviceMonitor.relabellings`        | Metrics RelabelConfigs to apply to samples before scraping.                                               | `[]`                     |
| `metrics.serviceMonitor.metricRelabelings`   | Metrics RelabelConfigs to apply to samples before ingestion.                                            | `[]`                     |
```

File: bitnami/redis/values.yaml [hunk 3]
```diff
@@ -1194,6 +1194,9 @@ metrics:
     ## @param metrics.serviceMonitor.relabellings Metrics relabellings to add to the scrape endpoint
     ##
     relabellings: []
+    ## @param metrics.serviceMonitor.metricRelabelings Metrics metricRelabelings to add to the scrape endpoint
+    ##
+    metricRelabelings: []
```
Comment (migruiz4) [inline bitnami/redis/values.yaml:1199]: Please also update the values `@param`:

```suggestion
    ## @param metrics.serviceMonitor.relabellings Metrics RelabelConfigs to apply to samples before scraping.
    ##
    relabellings: []
    ## @param metrics.serviceMonitor.metricRelabelings Metrics RelabelConfigs to apply to samples before ingestion.
    ##
    metricRelabelings: []
```

**Review by migruiz4 — COMMENTED** [review]
Hi @Vaproc,

Thank you very much for your contribution!

I left some comments below requesting some changes to the description of the values  `relabellings`(*) and `metricRelabelings` to make them easier to differentiate. Other than that your PR looks good to me!

**Comment by Vaproc** [comment]
Hi @migruiz4

Thank you for your suggestions. I've included them.

**Review by migruiz4 — APPROVED** [review]
Thank you, looks good to me!

**Pull request merged**